editor/plugins/gdextension/cpp_scons/template/addon/library_name.gdextension

```diff
@@ -0,0 +1,28 @@
+[configuration]
+
+entry_symbol = "__LIBRARY_NAME___library_init"
+compatibility_minimum = __GODOT_VERSION__
+reloadable = true
+
+[libraries]
+
+macos.debug = "bin/lib__LIBRARY_NAME__.macos.debug.framework"
+macos.release = "bin/lib__LIBRARY_NAME__.macos.release.framework"
+windows.debug.x86_64 = "bin/lib__LIBRARY_NAME__.windows.debug.x86_64.dll"
+windows.release.x86_64 = "bin/lib__LIBRARY_NAME__.windows.release.x86_64.dll"
```

Comment:
```suggestion
windows.debug.x86_64 = "bin/__LIBRARY_NAME__.windows.debug.x86_64.dll"
windows.release.x86_64 = "bin/__LIBRARY_NAME__.windows.release.x86_64.dll"
```

Since you obliged and used `SHLIBPREFIX` instead of `lib`, windows DLLs no longer have a `lib` prefix (which is correct idiomatic naming for the platform). 
I'm not sure if any other platforms are affected, too.

Reply:
Looks like Web, Android, and iOS also give `"lib"` for `env.subst('$SHLIBPREFIX')`.